### PR TITLE
feat: add more vendors for FFT benchmark

### PR DESCRIPTION
**Pull request by TomTaehoonKim**
# Description

Now, arkworks and bellman are benchmarked as well.

### CODE REVIEW

**Review by ashjeong — APPROVED** [review]
Some files seem to a bit repetitive and I think it'll continue to be more repetitive if more vendors are added. Maybe some fix for this will be required in the future. Otherwise, LGTM

**Review by Insun35 — APPROVED** [review]
LGTM

**Review by dongchangYoo — APPROVED** [review]
LGTM

File: benchmark/msm/msm_config.cc [hunk 1]
```diff
@@ -112,7 +114,7 @@ bool MSMConfig::Parse(int argc, char** argv,
     }
   }
 
-  base::ranges::sort(exponents_);
+  base::ranges::sort(exponents_);  // NOLINT
```
[the remark this answers is not in the document]
Comment (chokobole) [reply benchmark/msm/msm_config.cc:117]: I think this commit includes 2 changes

1. adding a missing header(absl/strings/substitute.h)
2. removing an unused header(algorithm), and here should add `NOLINT`

File: benchmark/fft/README.md [hunk 2]
```diff
@@ -12,37 +12,37 @@ CPU Caches:
 ## FFT
 
 ```shell
-bazel run -c opt --config halo2 --//:has_openmp --//:polygon_zkevm_backend --//:has_rtti --//:has_matplotlib //benchmark/fft:fft_benchmark -- -k 16 -k 17 -k 18 -k 19 -k 20 -k 21 -k 22 -k 23
+bazel run --config halo2 -c opt --//:has_openmp --//:has_rtti --//:has_matplotlib //benchmark/fft:fft_benchmark -- -k 16 -k 17 -k 18 -k 19 -k 20 -k 21 -k 22 -k 23 --vendor arkworks --vendor bellman --vendor halo2 --check_results
```
Comment (chokobole) [inline benchmark/fft/README.md:15]: This part contains removing `polygon_zkevm_backend`, and I think it's better to be squashed to the first commit!

**Review by chokobole — APPROVED** [review]
LGTM

**Pull request merged**